main_node:
  ros__parameters:
    robot_name: "jackal_ugv"
    imu_enabled: "true"
    camera_enabled: "false"
    stereo_camera_enabled: "false"
    lidar_enabled: "true"
    world_name: "new_indoor.world" #Leave empty to obtain no world
    sensors_topic:
      rgb_topic: "/camera/image_raw"
      depth_topic: "/camera/depth/image_raw"
      laser_topic: "/scan"
      imu_topic: "/imu/data"
      odom_topic: "/odom"
    laser_param:
      max_distance: 2.0 #Meters
      num_points: 60 #Maximum is 360
    depth_param:
      dist_cutoff: 5 #Meters
      width: 112
      height: 112
      show_image: False
    rgb_param:
      width: 640
      height: 480
      show_image: False
    policy: 'DDPG' # training policy
    policy_trainer: 'off-policy' # trainer off-policy / on-policy
    trainer_params: 'training_params.yaml'
    max_lin_vel: 1.0
    min_lin_vel: 0.0
    max_ang_vel: 3.0
    min_ang_vel: -3.0
    robot_type: "rectangular" # rectangular or circular
    robot_radius: 0.2 # Meters, If robot_type is cicrular
    robot_size: [0.508, 0.430] # Meters, Lenght, Width if robot_type is rectangular
    collision_tollerance: 0.05 # Meters, tollerance of collision added to robot_radius or robot_size
    sensor: 'applr' # lidar or rgb_camera or depth_camera
    visual_data: 'image'
    features: 12544
    channels: 1
    image_width: 112
    image_height: 112
    data_path: 'indoor.json'
    change_goal_and_pose: 5
    timeout_steps: 600
    
